extensions/wikia/UserActivity/UserActivityController.class.php

```diff
@@ -58,16 +54,16 @@ public function index() {
 	protected function formatItems( $items ) {
 		$flattened = [];
 		foreach ( $items as $sortKey => $contribItem ) {
-			$lastEditTS = wfTimestamp( TS_MW, $contribItem[ 'last_edit' ] );
+			$lastEditTS = wfTimestamp( TS_MW, $contribItem[ 'lastedit' ] );
+			unset($contribItem['lastedit']);
```

Comment:
What is the `'lastedit'` key vs `'lastEdit'` (line 61)? A :bug: magnet? :smiley_cat: